src/engine/objects/Zone.ts

```diff
@@ -54,7 +56,8 @@ export class Zone extends SimObject {
 
     const zoneMaterial = new THREE.MeshBasicMaterial(materialSpecs);
 
-    let fixtureShape;
+    // eslint-disable-next-line prefer-const
+    let fixtureShapes = [];
```

Comment:
i think this can be const, n.b. this is not the same as c++'s const, here it means i wont reassign this to another object. which since we only push things into this existing object is ok!